dotCMS/src/main/java/com/dotcms/experiments/business/result/VariantResultsBuilder.java

```diff
@@ -21,8 +24,7 @@ public class VariantResultsBuilder {
     private final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("MM/dd/yyyy")
```

Comment:
Not in current changes but this should be `static`.

Reply:
done https://github.com/dotCMS/core/pull/24363/commits/f8bdaeb099ab1d42329f204558e227f615692c29#diff-76e611bd72975f32c09111e2271a289c12747c71daa9b841b77f9160b29f419fR24